From 03a0144820fa1e5b832b8086a82d99b37b7152a1 Mon Sep 17 00:00:00 2001
From: Dave Stevenson <dave.stevenson@raspberrypi.com>
Date: Fri, 26 Jun 2020 15:53:44 +0100
Subject: [PATCH] media: bcm2835-unicam: Avoid gcc warning over {0} on
 endpoint

Older gcc versions object to = { 0 } initialisation if the first
elemtn in the structure is a substructure.

Use = { } to avoid this compiler warning.

Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.com>
---
 drivers/media/platform/bcm2835/bcm2835-unicam.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/media/platform/bcm2835/bcm2835-unicam.c
+++ b/drivers/media/platform/bcm2835/bcm2835-unicam.c
@@ -2586,7 +2586,7 @@ static const struct v4l2_async_notifier_
 static int of_unicam_connect_subdevs(struct unicam_device *dev)
 {
 	struct platform_device *pdev = dev->pdev;
-	struct v4l2_fwnode_endpoint ep = { 0 };
+	struct v4l2_fwnode_endpoint ep = { };
 	struct device_node *ep_node;
 	struct device_node *sensor_node;
 	unsigned int lane;
